neural_compressor/adaptor/torch_utils/smooth_quant.py

```diff
@@ -38,7 +38,9 @@ def move_input_to_device(input, device=torch.device('cpu')):
     elif isinstance(input, list) or isinstance(input, tuple):
         input_res, prev_size = [], None
         for inp in input:
-            if prev_size:
+            if isinstance(inp, tuple):
```

Comment:
can you add a test?

Reply:
yes, I will add a ut later. Thanks for reminding.